source/base/quadrature.cc

```diff
@@ -30,13 +31,8 @@ template <>
 Quadrature<0>::Quadrature (const unsigned int n_q)
```

Comment:
this is of course pre-existing, but out of curiosity: is the constructor that just  takes a number actually called anywhere? (In both 0d and higher dimensions?) I don't see why it would be useful. If it isn't (which should be easy to test), can you please open an issue that asks for it to be removed?